benchmark/clients.py

```diff
@@ -52,20 +52,22 @@ def create_requests_session(config: HttpClientConfig) -> requests.Session:
     return session
 
 
-def create_aioboto3_s3_client(config: HttpClientConfig, **kwargs):
+def create_aioboto3_s3_client(config: HttpClientConfig, region_name: str, **kwargs):
```

Comment:
Does it actually make a difference with botocore to pass this in?

Reply:
yes seems needed - https://github.com/geospatial-jeff/pyasyncio-benchmark/issues/35